dockerfiles/Dockerfile.apisix.rpm

```diff
@@ -3,6 +3,15 @@ ARG IMAGE_TAG="7"
 
 FROM ${IMAGE_BASE}:${IMAGE_TAG}
 
+RUN if [[ $(rpm --eval '%{centos_ver}') == "8" ]]; then \
```

Comment:
Is there a way to share the same code in three places?

Reply:
I afraid there is no such a way to share the same code **<ins>within</ins>** the `Dockerfile`.

But there are some trick can achieve the goal as well:
1. pre-build a repo patch applied image, then refer it in final build Dockerfile
2. add a utils script for this scenario **<ins>only</ins>**, then we can reuse the container filesystem layer cache